spec/features/shelter/shelter_update_spec.rb

```diff
@@ -42,4 +42,33 @@
     expect(current_path).to eq("/pets")
   end
 
+  it "has a flash message indicating which fields are missing" do
+
+    click_button "Update Shelter"
+    fill_in :name, with: nil
```

Comment:
Prefills, so we basically just have to delete it to test if it doesn't have the information or not